src/Core/Components/Slider/FluentSlider.razor.cs

```diff
@@ -0,0 +1,130 @@
+// ------------------------------------------------------------------------
+// MIT License - Copyright (c) Microsoft Corporation. All rights reserved.
+// ------------------------------------------------------------------------
+
+using System.Diagnostics.CodeAnalysis;
+using Microsoft.AspNetCore.Components;
+using Microsoft.AspNetCore.Components.Web;
+using Microsoft.FluentUI.AspNetCore.Components.Extensions;
+
+namespace Microsoft.FluentUI.AspNetCore.Components;
+
+/// <summary>
+/// FluentSlider component, a slider control that allows users to select from a range of values.    
+/// </summary>
+/// <typeparam name="TValue"></typeparam>
+public partial class FluentSlider<TValue> : FluentInputBase<TValue>
+    where TValue : struct, IComparable<TValue>
+{
+    /// <summary>
+    /// Gets or sets the size for the slider.
+    /// </summary>
+    [Parameter]
+    public SliderSize? Size { get; set; }
+
+    /// <summary>
+    /// Gets or sets the slider's minimal value.
+    /// </summary>
+    [Parameter]
+    public TValue? Min { get; set; }
+
+    /// <summary>
+    /// Gets or sets the slider's maximum value.
+    /// </summary>
+    [Parameter]
+    public TValue? Max { get; set; }
+
+    /// <summary>
+    /// Gets or sets the slider's step value.
+    /// </summary>
+    [Parameter]
+    public TValue? Step { get; set; }
+
+    /// <summary>
+    /// Gets or sets the orientation of the stacked components.
+    /// </summary>
+    [Parameter]
+    public Orientation Orientation { get; set; }
+
+    /// <summary>
+    /// Gets or sets the selection mode.
+    /// </summary>
+    [Parameter]
+    public SliderMode? Mode { get; set; }
+
+    /// <summary>
+    /// Gets or sets the content to be rendered inside the component.
+    /// </summary>
+    [Parameter]
+    public RenderFragment? ChildContent { get; set; }
```

Comment:
What is the goal of this property? I tried this code, but without success.
```xml
<fluent-slider>Hello</fluent-slider>
```

I will remove this property **ChildContent**.